Networking/Networking/Model/Product/ProductType.swift

```diff
@@ -14,6 +14,21 @@ public enum ProductType: Codable, Hashable, GeneratedFakeable {
     case composite
     case booking
     case custom(String) // in case there are extensions modifying product types
+
+    public static var allCases: [ProductType] {
+        return [
+            .simple,
+            .grouped,
+            .affiliate,
+            .variable,
+            .subscription,
+            .variableSubscription,
+            .bundle,
+            .composite,
+            .booking,
+            .custom("exampleCustomType") // hardcoded to be able to include .custom in CaseIterable.
+        ]
+    }
```

Comment:
This is currently used only for tests, so how about moving the `CaseIterable` conformance to the test file only? It doesn't make sense to keep `exampleCustomType` in production code IMO.

Reply:
I hadn't thought of that, good point! https://github.com/woocommerce/woocommerce-ios/pull/14068/commits/fa038ce32abe1f6ce6a76444f054613b3bd41b70